nss-day-cohort-51/nutshell-vampire-brood
# Mw fix updating task files

# Description Cleanup ApplicationViews. Remove duplicates removed duplicate lines and referenced all objects.

## Type of change

- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
You have the ability to view all application components and their respect create, edit, delete buttons.
Access each component from the Navbar dApplicationViews Events from the Events link in Nav
Tasks should have Material UI component styling

# Checklist:
